cmd/snap-confine/snap-confine.apparmor.in

```diff
@@ -68,7 +68,7 @@
     /etc/udev/udev.conf r,
     /sys/**/uevent r,
     /usr/lib/snapd/snap-device-helper ixr, # drop
-    /lib/udev/snappy-app-dev ixr, # drop
+    /{,usr/}lib/udev/snappy-app-dev ixr, # drop
```

Comment:
Are we still shipping this in new installs?

Reply:
We don't ship it for new installs. I *think* we might need it for classic re-exec cases but I'm not fully sure.